client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
Definitely! 
I think it may be helpful to add some helpers for this but I rather that be in a different pr